### DISCUSSION TITLE
Two or more presets on my vault

### DISCUSSION THREAD

**Discussion by radgzx** [Akifyss/obsidian-border#92] [Q&A]
Hi, I would like to know if I can have more than one preset imported into my vault at the same time, and if so, how can I do it?

**Comment by Akifyss** [comment]
Hello, you can only import up to two presets (one for light mode and one for dark mode). Just copy the code of the presets and import them in the style settings. However, please note that it is recommended to reset the corresponding options to default before importing the presets.

**Comment by radgzx** [comment]
Thank you!
